pkg/apis/bundle/v1alpha1/builder_types.go

```diff
@@ -92,6 +92,9 @@ type TemplateFileSet struct {
 	// to Go Templates during build if left unspecified.
 	TemplateType TemplateType `json:"templateType,omitempty"`
 
+	// UseSafeYAMLTemplater determines whether to use SafeYAMLTemplating.
+	UseSafeYAMLTemplater *bool `json:"useSafeYamlTemplater,omitempty"`
```

Comment:
Do any of these UseSafeYAMLTemplater options override each other depending on where they're set? If so we should make the overriding/inheritance behavior clear in doc comments/actual docs.

Reply:
Good point. Added some laungage that this is a default and doesn't override the downstream template settings.